### ISSUE TITLE
Hindi font not working on copy paste

### ISSUE THREAD

**Issue by saurabhdaware** [saurabhdaware/text-to-handwriting#76]
**Describe the bug**
The font works when you type it with the keyboard but does not work when you copy-paste from a different website. This is probably happening because of different Unicode so the font taken from Wikipedia, has different characters than the characters it get from typing.

Not really sure how we can fix this

**To Reproduce**
1. Go to Wikipedia and copy any hindi font
2. Paste it in textarea. The handwritten font will be replaced with a normal text

**Comment by prafulla-codes** [comment]
I'll try to work on this

**Comment by saurabhdaware** [comment]
Yeah sure @Pika1998 not quite sure how to solve this though so we will have to figure it out

**Comment by prafulla-codes** [comment]
I did try to debug it, but it seems this problem is unsolvable because I think the general characters used in Hindi_Font and the font used in Wikipedia are made up of different characters

Here is a possible solution
We use an intermediary translation tool (Perhaps an API which translates our copied clipboard text to English & then we try to convert that to some general hindi font which supports all characters typed using English) I hope it makes sense lol.

**Comment by saurabhdaware** [comment]
yeah that makes sense. Will need to find something that does that for us
